src/vivarium_public_health/population/add_new_birth_cohorts.py

```diff
@@ -125,14 +110,48 @@ def add_new_birth_cohort(self, event):
                                       'age_end': 0,
                                   })
 
-    def _get_birth_rate(self, year):
+    @staticmethod
+    def get_crude_birth_rate(builder):
+        population_data = load_population_structure(builder)
+        exit_age = builder.configuration.population.to_dict().get('exit_age', None)
+        if exit_age:
+            if builder.configuration.population.age_end != exit_age:
+                raise ValueError('If you specify an exit age, the initial population age end must be the same '
+                                 'for the crude birth rate calculation to work.')
+            cut_bin = population_data[(population_data.age_group_start < exit_age)
+                                      & (population_data.age_group_end >= exit_age)]
```

Comment:
I think you want this to be just > not >= or you want line 125 to be < and not <=. If you end up in the situation where you have a bin with age_group_end == exit_age, I think this will include it twice - once here and once in line 125